### PR TITLE
Fix: potential SQL injection bug in Number Tools

### LINKED ISSUE #684: SQL Injection vulnerability
In the Tools > Number:

> The `orderby` and `order` parameters are not properly sanitized in the SQL query, resulting in a potential injection vulnerability in the database. Since it utilizes the GET request method, a malicious user can exploit this by creating a specially crafted HTML page and then sending a request to the vulnerable endpoint (similar to CSRF attack).

### PR DESCRIPTION
**Pull request by alexmigf**
closes #684

### CODE REVIEW

File: includes/tables/class-wcpdf-number-store-list-table.php [hunk 1]
```diff
@@ -193,14 +193,15 @@ public function get_search() {
...
 		$results                        = array();
-		$paged                          = $this->get_paged();
+		$paged                          = $this->get_paged( $request );
 		$offset                         = $this->per_page * ( $paged - 1 );
-		$search                         = $this->get_search();
-		$table_name                     = isset( $_GET['table_name'] ) ? sanitize_text_field( $_GET['table_name'] ) : null;
-		$order                          = isset( $_GET['order'] ) ? sanitize_text_field( $_GET['order'] ) : 'DESC';
-		$orderby                        = isset( $_GET['orderby'] ) ? sanitize_text_field( $_GET['orderby'] ) : 'id';
+		$search                         = $this->get_search( $request );
+		$table_name                     = isset( $request['table_name'] ) ? sanitize_text_field( $request['table_name'] ) : null;
```
Comment (MohamadNateqi) [inline includes/tables/class-wcpdf-number-store-list-table.php:202]: Do you think it's better not to take the table name from users? I don't see why we need to have this option here.

File: includes/tables/class-wcpdf-number-store-list-table.php [hunk 2]
```diff
@@ -193,14 +193,15 @@ public function get_search() {
...
+		$order                          = isset( $request['order'] ) && in_array( $request['order'], array( 'DESC', 'ASC' ) ) ? sanitize_text_field( $request['order'] ) : 'DESC';
+		$orderby                        = isset( $request['orderby'] ) && in_array( $request['orderby'], array( 'id' ) ) ? sanitize_text_field( $request['orderby'] ) : 'id';
```
Comment (MohamadNateqi) [inline includes/tables/class-wcpdf-number-store-list-table.php:204]: Same for this, since it's always `id`.

Comment (alexmigf) [reply includes/tables/class-wcpdf-number-store-list-table.php:204]: For now it is, but it's ready to accept others.

File: includes/tables/class-wcpdf-number-store-list-table.php (hunk 1 above)
Comment (alexmigf) [reply includes/tables/class-wcpdf-number-store-list-table.php:202]: Because you can select the table from the UI.

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

File: includes/tables/class-wcpdf-number-store-list-table.php (hunk 1 above)
Comment (alexmigf) [reply includes/tables/class-wcpdf-number-store-list-table.php]: Please check my last commit.

**Review by MohamadNateqi — APPROVED** [review]
Great change!
LGTM!

**Pull request merged**